infra/resources/_modules/function_apps/locals.tf

```diff
@@ -13,8 +13,33 @@ locals {
       FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
       FETCH_KEEPALIVE_TIMEOUT             = "60000"
 
-      // CosmosDB connection
-      COSMOS_DB_ENDPOINT = var.cosmos_db_endpoint
-    }
+      CosmosDbConnectionString = "AccountEndpoint=${var.cosmos_db_endpoint};AccountKey=${var.cosmos_db_key};"
+      CosmosDbDatabaseName     = var.cosmos_database_names[0]
+
+      FederationEntityBasePath         = "https://io-d-wallet-it.azurewebsites.net"
+      FederationEntityOrganizationName = "PagoPa S.p.A."
+      FederationEntityHomepageUri      = "https://io.italia.it"
+      FederationEntityPolicyUri        = "https://io.italia.it/privacy-policy/"
+      FederationEntityTosUri           = "https://io.italia.it/privacy-policy/"
+      FederationEntityLogoUri          = "https://io.italia.it/assets/img/io-it-logo-blue.svg"
+      FederationEntityTrustAnchorUri   = "https://demo.federation.eudi.wallet.developers.italia.it"
+      IosBundleIdentifier              = "it.pagopa.app.io"
+      IosTeamIdentifier                = "DSEVY6MV9G"
+      AndroidBundleIdentifier          = "it.pagopa.app.io"
+      AndroidCrlUrl                    = "https://android.googleapis.com/attestation/status"
+      AndroidPlayIntegrityUrl          = "https://www.googleapis.com/auth/playintegrity"
+      AndroidPlayStoreCertificateHash  = "INVALID"
```

Comment:
@LazyAfternoons Can you give us the correct value?

Reply:
The value for `io-app`, extracted directly from the APK and matched on Google Play Console is: 
`7D:E4:FE:3E:AA:E0:83:FF:CD:8B:07:03:01:E8:65:02:D3:F1:EA:D1:DD:66:AC:14:2B:AD:7C:54:47:55:4F:82`

![image](https://github.com/pagopa/io-wallet/assets/12371664/d058eaaf-e32f-45ef-95ca-ca1011030188)

Reply:
I think it should be converted first in HEX or base64 string. @LazyAfternoons can you check how it comes from the attestation?

Reply:
According to our [DR](https://pagopa.atlassian.net/wiki/spaces/SIW/pages/921436266/IO-WALLET-DR-0007+Play+Integrity+API) and the [official documentation](https://developer.android.com/google/play/integrity/verdicts#application-integrity-field), the SHA-256 is an URL-safe base64 encoded string: 
```json
appIntegrity: {
  // PLAY_RECOGNIZED, UNRECOGNIZED_VERSION, or UNEVALUATED.
  appRecognitionVerdict: "PLAY_RECOGNIZED"
  // The package name of the app.
  // This field is populated iff appRecognitionVerdict != UNEVALUATED.
  packageName: "com.package.name"
  // The sha256 digest of app certificates (base64-encoded URL-safe).
  // This field is populated iff appRecognitionVerdict != UNEVALUATED.
  certificateSha256Digest: ["6a6a1474b5cbbb2b1aa57e0bc3"]
  // The version of the app.
  // This field is populated iff appRecognitionVerdict != UNEVALUATED.
  versionCode: "42"
}
```

Reply:
Ok so it should be: `feT-Pqrgg__NiwcDAehlAtPx6tHdZqwUK618VEdVT4I`